cmd/generate-config/config/config-openapi-spec.json

```diff
@@ -59,6 +60,39 @@
             }
           }
         },
+        "featureGates": {
+          "type": "object",
+          "required": [
+            "customNoUpgrade",
+            "featureSet"
+          ],
```

Comment:
These two fields are marked as required but default values and validations can handle empty values.

Reply:
I think this may not be an issue because existing elements of the config also include 
a populated `required` array: 
https://github.com/openshift/microshift/blob/ad6b2f2e8b4543350095ccdc7116e19e3514ed7f/cmd/generate-config/config/config-openapi-spec.json#L3-L16